.sha256sum

```diff
@@ -1,2 +1,2 @@
 468fa0da0a50d50640ec57843ad288af343128b39f5bf23e76e4e336580883d4  swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json
-5369bd05f66fb79c8bd0836a980eea438974cc94cb8a073104ee218da8612602  swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-09-01-preview/redhatopenshift.json
+c323c84befa5ea11da50a2407050abed6540ea01e796720bc2241604ce80567c  swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-09-01-preview/redhatopenshift.json
```

Comment:
this is reverting to the previous version prior to FIPS in preview-api https://github.com/Azure/ARO-RP/commit/63382712ce4a84ff6c8e1bd02aa9a7c6bb5e393a#diff-93d1c0fc746e9585efa95894ca536f43a0ba7c2b0643e8ade3612e06b58ad3f2